### ISSUE TITLE
Not working with tfrecords

### ISSUE THREAD

**Issue by darthgera123** [calmisential/TensorFlow2.0_SSD#16]
Hi, instead of reading from text files, I am instead trying to read from tfrecords which i created and returned images and labels in the exact same format. However on training I am getting this error in ground_truth.py
```
RuntimeWarning: divide by zero encountered in log
  g_h = np.log(np.abs(g_h) / d_h)
```
i used abs with g_h as it was giving negative value at times. any workaround
